package.json

```diff
@@ -18,15 +18,16 @@
     "test-browser-chrome": "npm run small-build && ( browserify test/*.js | tape-run --browser chrome | faucet )",
     "test-browser-safari": "npm run small-build && ( browserify test/*.js -t [ babelify --presets [ es2015 ] ] | tape-run --browser safari | faucet )",
     "test-browser-firefox": "npm run small-build && ( browserify test/*.js  | tape-run --browser firefox | faucet )",
-    "test-travis": "npm run full-test && npm run quick-build && istanbul cover tape test/*.js test/typescript/typescript-tests.js",
+    "test-travis": "npm run full-test && npm run quick-build && istanbul cover tape test/*.js test/typescript/typescript-tests.js && npm run size",
     "test-flow": "node_modules/.bin/flow check",
     "test-webpack": "node scripts/webpack-regression-tests.js",
     "coverage": "npm run quick-build && npm run build-tests && istanbul cover tape test/*.js test/typescript/typescript-tests.js",
     "build-tests": "npm run build-typescript-tests && npm run build-babel-tests",
     "build-typescript-tests": "tsc -p test/typescript",
     "build-babel-tests": "babel test/babel/babel-tests.js -o test/babel-tests.js",
     "use-minified": "cp lib/mobx.min.js lib/mobx.js",
-    "lint": "tslint -c tslint.json src/*.ts src/types/*.ts src/api/*.ts src/core/*.ts src/utils/*.ts"
+    "lint": "tslint -c tslint.json src/*.ts src/types/*.ts src/api/*.ts src/core/*.ts src/utils/*.ts",
+    "size": "size-limit 19KB lib/mobx.js"
```

Comment:
the 19KB is quite arbitrary, why not 20? I assume the goal is to check whether the size is not unexpectedly blown up?